domain-xample/xample-workflows/src/main/java/gov/va/vro/routes/xample/DbHelper.java

```diff
@@ -60,7 +60,20 @@ public ContentionEventEntity saveContentionEvent(final BieMessagePayload bieMess
     contentionEventEntity.setDiagnosticTypeCode(bieMessagePayload.getDiagnosticTypeCode());
     contentionEventEntity.setContentionClassificationName(
         bieMessagePayload.getContentionClassificationName());
-    contentionEventEntity.setOccurredAt(convertTime(bieMessagePayload.getOccurredAt()));
+    contentionEventEntity.setOccurredAt(convertTime(bieMessagePayload.getEventTime()));
+    contentionEventEntity.setDateAdded(convertTime(bieMessagePayload.getDateAdded()));
+    contentionEventEntity.setDateCompleted(convertTime(bieMessagePayload.getDateCompleted()));
+    contentionEventEntity.setDateUpdated(convertTime(bieMessagePayload.getDateUpdated()));
+    contentionEventEntity.setActorStation(bieMessagePayload.getActorStation());
+    contentionEventEntity.setAutomationIndicator(bieMessagePayload.getAutomationIndicator());
```

Comment:
Can we not chain these set statements? that is

`
contentionEventEntity.setOccurredAt(...)
.setDateAdded(...)
.
.
.
`